### PR TITLE
Quizzes questions

### PR DESCRIPTION
**Pull request by gopalbbharadva**
#### Features
- Questions and answers
- Result page
- Total score

Live Link: https://deploy-preview-3--stellar-malasada-714a70.netlify.app/

P.S: Start checking from 6th commit

### CODE REVIEW

File: src/contexts/AuthContext.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  const signUp = (auth: Auth, email: string, password: string) => {
+    return createUserWithEmailAndPassword(auth, email, password);
+  };
+
+  const logOut = async () => {
+    try {
+      await signOut(auth);
+      navigate("/");
+    } catch (error) {
+      console.log(error);
```
Comment (manojasarsa) [inline src/contexts/AuthContext.tsx:50]: You can use `console.error` here

**Review by manojasarsa — APPROVED** [review]
Code looks clean and readable.
GTG!

File: src/contexts/QuizContext.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      try {
+        const res = await getDocs(quizzesRef);
+        quizDispatch({
+          type: "GET_QUIZZES",
+          payload: {
+            quizzes: res.docs.map((item) => ({ ...item.data(), id: item.id })),
+          },
+        });
+      } catch (error) {
+        console.log(error);
```
Comment (mimansha-swarup) [inline src/contexts/QuizContext.tsx:41]: using console.error will be better

File: src/contexts/QuizContext.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
+import {  getDocs } from "firebase/firestore";
+import { createContext, useContext, useEffect, useReducer } from "react";
+import { categoryRef, quizzesRef } from "../firebase/FirebaseConfig";
+import { initialState, QuizReducer } from "../reducers/QuizReducer";
+import { QuizContextProps } from "../types/ContextTypes/QuizContext.types";
+import { ChildrenProps } from "../types/ReactEvents.types";
```
Comment (mimansha-swarup) [inline src/contexts/QuizContext.tsx:6]: hey you can use barrel export here

**Review by mimansha-swarup — APPROVED** [review]

**Pull request merged**